presto-orc/src/main/java/com/facebook/presto/orc/reader/StreamReader.java

```diff
@@ -164,4 +164,15 @@ void startRowGroup(InputStreamSources dataStreamSources)
     void close();
 
     long getRetainedSizeInBytes();
+
+    // Returns true if the reader recursively contains a reader that depends
```

Comment:
these comment should use javadoc style; the text can be simplified for clarity

```
    /**
     * Returns true if the reader must flush accumulated results before processing
     * next row group.
     * 
     * This would happen if results are encoded using stripe or row group dictionaries. 
     *
     * @param newStripe boolean indicating whether the next row group comes from a new stripe.
     */
    default boolean mustExtractValuesBeforeNextRowGroup(boolean newStripe)
```